recipe/meta.yaml

```diff
@@ -10,46 +11,61 @@ source:
   sha256: 8c4a90bb72f750eff0a1df24d2370ec1f5584daccf5f2a477ed77a2e72850eba
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 2
 
-requirements:
-  host:
-    - numpy >=1.16.0
-    - pandas >=0.25.0
-    - pip
-    - python >=3.7
-    - pytz
-    - requests
-    - scipy >=1.2.0
-    - h5py
-    - numba
-    - ephem
-    - netcdf4
-    - cftime >=1.1.1
-    - statsmodels
-    - importlib-metadata
-    - setuptools_scm >=6.2
-  run:
-    - numpy >=1.16.0
-    - pandas >=0.25.0
-    - pip
-    - python >=3.7
-    - pytz
-    - requests
-    - scipy >=1.2.0
-    - h5py
-    - numba
-    - ephem
-    - netcdf4
-    - cftime >=1.1.1
-    - statsmodels
-    - importlib-metadata
+outputs:
+  - name: {{ name }}
+    build:
+      noarch: python
+      script: python -m pip install . --no-deps -vv
+    requirements:
+      host:
+        - numpy >=1.16.0
+        - pandas >=0.25.0
+        - pip
+        - python >=3.7
+        - pytz
+        - requests
+        - scipy >=1.2.0
+        - h5py
+        - numba
+        - ephem
+        - netcdf4
+        - cftime >=1.1.1
+        - statsmodels
+        - importlib-metadata
+        - setuptools_scm >=6.2
+      run:
+        - numpy >=1.16.0
+        - pandas >=0.25.0
+        - pip
+        - python >=3.7
+        - pytz
+        - requests
+        - scipy >=1.2.0
+        - h5py
+        - numba
+        - ephem
+        - netcdf4
+        - cftime >=1.1.1
+        - statsmodels
+        - importlib-metadata
+    test:
+      imports:
+        - pvlib
 
-test:
-  imports:
-    - pvlib
+  - name: pvlib-python
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.7
+        - pip
```

Comment:
Probably doesn't matter, but the importlib-metadata and typing-extension recipes don't include `host` requirements for the secondary output.

Reply:
Good catch. Holdover from failed experiment while trying to make it work with `{{ PYTHON }}`. Better to keep it simple and remove it if doesn't actually help us.